src/__experimental__/components/search/search.style.js

```diff
@@ -81,16 +81,23 @@ export const StyledSearchButton = styled.div`
   display: inline-flex;
   border-bottom: 1px solid transparent;
   &&& ${StyledButton} {
-    ${({ theme }) => `background-color: ${theme.colors.primary};`}
-    ${({ theme }) => `border-color: ${theme.colors.primary};`}
+    ${({ theme }) => `
+    background-color: ${theme.colors.primary};
+    border-color: ${theme.colors.primary};
+    :hover {
+      background: ${theme.colors.secondary};
+    }
+    `}
     width: 43px;
     height: 43px;
     margin: 0px 0px;
     padding-bottom: 3px;
     transition: border 0.2s ease, background 0.2s ease;
     :focus {
      z-index: 10;
-   }
+    }
```

Comment:
looks like a wrong indentation

Reply:
agree with @damianOfKyan  and you can remove the empty lines as well